### PR TITLE
Ayush 🍉

### PR DESCRIPTION
**Pull request by chromo4**
Added my name to contributing in that list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,17 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore-start -->
 <!-- markdownlint-disable -->
+
```
Comment (alisolanki) [inline README.md:38]: Please avoid adding extra spaces

File: README.md [hunk 2]
```diff
@@ -35,9 +35,17 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
...
 <table>
     <tbody>
         <tr>
+            <td align="center">
```
Comment (alisolanki) [inline README.md:42]: Please add your name to the end of the list

**Review by alisolanki — CHANGES_REQUESTED** [review]
Please do the needful

File: README.md (hunk 1 above)
Comment (chromo4) [reply README.md:38]: done please check, sorry if I done the mistake I am new to this

File: README.md (hunk 2 above)
Comment (chromo4) [reply README.md]: please check

**Pull request merged**